maas-schemas/test/schemas/maas-backend/products/valid-provider.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "providerName",
+  "agencyId": "careco-cug",
```

Comment:
Replace `agencyId` value with an anonymized one.

Reply:
There is `ontrack` below as well.

Reply:
thanks guys! changed